### ISSUE TITLE
Font name issue in custom font

### ISSUE THREAD

**Issue by onegladiator4u** [eliranwong/UniqueBible#211]
@otseng When I selected my font named 'NotoSerifMalayalam-Regular.ttf' through modify database the font didn't work.
Later it worked only after changing the name of the file in font folder from 'NotoSerifMalayalam-Regular.ttf' to 'Noto Serif Malayalam.ttf'. 
I tried one more language I know. If the file name is different from the name font file shows in the title bar it doesn't work. 
![image](https://user-images.githubusercontent.com/48617390/109410262-392e9280-79bf-11eb-9cd7-3dba97704317.png)
I think the file name shouldn't be the value that goes to FontName in the database value. I'm not sure.

**Comment by otseng** [comment]
Made some more changes to try to address this issue.  Now the name of the font file shouldn't matter.  

The reason renaming the font file worked for you is it was loading the name from your installed fonts, not from the font file.

Also, I deleted the Malayalam font files, so you'll need to put them back in.  

Not sure if it completely fixes the issue, so feel free to create another issue.

**Comment by onegladiator4u** [comment]
> The reason renaming the font file worked for you is it was loading the name from your installed fonts, not from the font file.

Oh ok. Got it.
My issue was solved. I don't want anyone else to have the problem. Hopefully it won't .

Can I suggest you to add the info on the values that can be used for font size in changelog or something. Yesterday I had only 2 values x-small and x-large. I had to search the net for other values. I tried numbers it didn't work until adding px to it. Just a suggestion. Thanks for your work.

**Comment by otseng** [comment]
Updated the wiki with some example font size values:

https://github.com/eliranwong/UniqueBible/wiki/Using-custom-fonts-with-Bibles

**Comment by otseng** [comment]
Your issue made me think that it's easier just to use a built-in font instead of providing a ttf file.  I've created an issue to support both builtin fonts and ttf files:

https://github.com/eliranwong/UniqueBible/issues/213

**Comment by onegladiator4u** [comment]
@otseng Yeah. That sounds better..
